### PR TITLE
A lot of our tests require sudo

### PR DESCRIPTION
**Pull request by jaym**
We have enough travis capacity now to have longer running tasks

### CODE REVIEW

**Comment by lamont-granquist** [comment]
how did we get more travis capacity?

**Comment by jaym** [comment]
We pay for it

**Comment by lamont-granquist** [comment]
so you got "sudo: bundle: command not found" as an error

and you get a yellow "warning: this is running on our legacy infrastructure, read how to upgrade" banner warning from travis.

**Comment by jaym** [comment]
There are a couple of failing tests that will probably have to be disabled in travis. Is this something @chef/client-core is interested in? Tests will take longer to start since we will not be using the container infrastructure.

**Comment by danielsdeleo** [comment]
Is it possible to combine the two and have one matrix entry just for root and one for not-root?

**Comment by jaym** [comment]
Running in the new infrastructure that allows docker + sudo. Not sure what's up with the latest bundler, pinned to a working version

**Comment by jaym** [comment]
A lot more failing tests now, perhaps the next stuff won't work for us :(

**Comment by lamont-granquist** [comment]
Bunch of networking stuff.  Is the new docker infra trying to save us from ourselves by not allowing networking?

**Comment by lamont-granquist** [comment]
try a rebase against current master?

**Comment by thommay** [comment]
@jaym what do you want to do with this?

**Comment by lamont-granquist** [comment]
@jaym i just wrote a bunch of functional specs for dpkg_package which require root, would love to see this one rebased and see if it works or not...

**Comment by jaym** [comment]
Let's give it a shot, IIRC, there were issues with ipv6 tests

**Comment by lamont-granquist** [comment]
LGTM :+1:

**Comment by lamont-granquist** [comment]
@chef/client-core review?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -121,6 +121,7 @@ module Shell
 
   config.filter_run_excluding :skip_appveyor => true if ENV["APPVEYOR"]
   config.filter_run_excluding :appveyor_only => true unless ENV["APPVEYOR"]
+  config.filter_run_excluding :skip_travis => true if ENV["TRAVIS"]
```
Comment (tyler-ball) [inline spec/spec_helper.rb]: Can you add a comment here explaining _why_ we need to skip certain tests on Travis?

**Comment by tyler-ball** [comment]
:+1:

**Pull request merged**